src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,149 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> tail;
+    private Node<T> head;
+    private int size;
+
     @Override
     public void add(T value) {
+        add(value, size);
```

Comment:
I you can see you passed size as index, so only 26th line will be applied for this case. Extracting 27th to 29 to separated method let you call linkTail just from here